mmcv/visualization/image.py

```diff
@@ -27,15 +31,15 @@ def imshow(img, win_name='', wait_time=0):
         ret = cv2.waitKey(wait_time)
 
 
-def imshow_bboxes(img,
-                  bboxes,
-                  colors='green',
-                  top_k=-1,
-                  thickness=1,
-                  show=True,
-                  win_name='',
-                  wait_time=0,
-                  out_file=None):
+def imshow_bboxes(img: Union[str, np.ndarray],
+                  bboxes: Union[list, np.ndarray],
+                  colors: Union[str, tuple, int, np.ndarray] = 'green',
```

Comment:
```suggestion
                  colors: Union[Color, str, tuple, int, np.ndarray] = 'green',
```

Reply:
Please help update the type in https://github.com/open-mmlab/mmcv/pull/1946/files#diff-7ab945ee4fc4c1df5be4905c547ba7fb381e381a04876b1968444b4689c2df9bR48